libs/ngworker/lumberjack/tsconfig.spec.json

```diff
@@ -5,6 +5,9 @@
     "module": "commonjs",
     "types": ["jest", "node"]
   },
+  "typeAcquisition": {
+    "enable": true,
+    "include": ["jest"]},
```

Comment:
question: Why is this needed? According to [the TypeScript docs](https://www.typescriptlang.org/tsconfig#typeAcquisition), this is only relevant for JavaScript.

Reply:
Nothing, me trying to fix that the Jest name wasn't available. But you fixed in another commit.